app/server/src/main/scala/org/bitcoins/server/ServerJsonModels.scala

```diff
@@ -483,7 +484,71 @@ object OpReturnCommit extends ServerJsonModels {
             s"Bad number of arguments: ${other.length}. Expected: 3"))
     }
   }
+}
+
+// Oracle Models
+
+case class CreateEvent(label: String, outcomes: Vector[String])
```

Comment:
Why do you call it `label` rather than `eventName`? I prefer the latter if you are indifferent.

Reply:
Done in #2094